web_src/js/features/repo-issue.js

```diff
@@ -287,6 +287,20 @@ export function initRepoPullRequestMergeInstruction() {
   });
 }
 
+export function initRepoPullRequestAllowEditsFromMaintainers() {
+  $('#allow-edits-from-maintainers input').on('change', function () {
+    let url = '';
+    if ($(this).is(':checked')) { // allow edits
+      url = `${window.location}/allow_edits`;
+    } else { // disallow edits
+      url = `${window.location}/disallow_edits`;
```

Comment:
This will fail if there is a query string or hash in the current URL. It should specify the URL based on variables instead, other ajax requests can serve as example.